CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.1] - 2021-08-26
+### Added
+- all input `state` type support
+### Changes
+- `make_prob_dist` modified
+- new examples
```

Comment:
I think it would be better if move this line and the 2 lines below to the `### Added` section

Reply:
Yes you're right.
That issue occurred when merging.